apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.CommonTesting/SpannerTestDatabase.cs

```diff
@@ -85,15 +88,23 @@ private SpannerTestDatabase(string projectId)
             }
             NoDbConnectionString = builder.ConnectionString;
             ConnectionString = builder.WithDatabase(SpannerDatabase).ConnectionString;
```

Comment:
Maybe remove this line since ConnectionString is being set again one line down?

Reply:
Whoops, yes. Not sure why that was still there.